lib/PEX.ts

```diff
@@ -138,7 +138,7 @@ export class PEX {
    */
   public selectFrom(
     presentationDefinition: IPresentationDefinition,
-    verifiableCredentials: OriginalVerifiableCredential[],
+    verifiableCredentials: (IVerifiableCredential | JwtDecodedVerifiableCredential | CompactJWT)[],
```

Comment:
This is literally what OriginalVerifiableCredential is

Reply:
fixed